<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard::CDashboardWidget</class>
 <widget class="QWidget" name="Dashboard::CDashboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGraphicsView" name="dashboardView">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="Dashboard::CDashboardComboBox" name="pgmsComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>53</height>
        </size>
       </property>
       <property name="maxVisibleItems">
        <number>5</number>
       </property>
       <property name="maxCount">
        <number>5</number>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>53</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>53</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/HimalayaImages/IconPushButton/IconPushButton-Enabled.png);</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../Platform/Gui/Components/MainMenu/Resource/MainMenu.qrc">
         <normaloff>:/HimalayaImages/Icons/Dashboard/Operation/Operation_Start_Resume.png</normaloff>:/HimalayaImages/Icons/Dashboard/Operation/Operation_Start_Resume.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>53</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>53</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/HimalayaImages/IconPushButton/IconPushButton-Enabled.png);</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/HimalayaImages/Icons/Dashboard/Operation/Operation_Abort.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>53</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>42</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>42</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="labelUnlock">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Unlock</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelRetort">
         <property name="text">
          <string>Retort:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="MainMenu::CSliderControl" name="retortSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelLock">
         <property name="text">
          <string>Lock</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainMenu::CSliderControl</class>
   <extends>QSlider</extends>
   <header>MainMenu/Include/SliderControl.h</header>
  </customwidget>
  <customwidget>
   <class>Dashboard::CDashboardComboBox</class>
   <extends>QComboBox</extends>
   <header>Dashboard/Include/DashboardComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../Platform/Gui/Components/MainMenu/Resource/MainMenu.qrc"/>
 </resources>
 <connections/>
</ui>
